matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerExporter.java

```diff
@@ -274,7 +394,11 @@ private void createTieLines(Network network, MatpowerModel model, Context contex
         for (TieLine l : network.getTieLines()) {
             Terminal t1 = l.getDanglingLine1().getTerminal();
             Terminal t2 = l.getDanglingLine2().getTerminal();
-            createMBranch(t1, t2, l.getR(), l.getX(), l.getB1(), l.getB2(), context).ifPresent(model::addBranch);
+            createMBranch(t1, t2, l.getR(), l.getX(), l.getB1(), l.getB2(), context)
```

Comment:
Typically in our network we can met:
````
    <currentLimits1 permanentLimit="786"/>
    <currentLimits2 permanentLimit="786">
      <temporaryLimit name="IT20" acceptableDuration="1200" value="1010"/>
      <temporaryLimit name="IT10" acceptableDuration="600" value="1177"/>
      <temporaryLimit name="IT1" acceptableDuration="60"/>
    </currentLimits2>
````
Temporary limits are only located side 2.

Reply:
Fixed